recipes/bm3d/meta.yaml

```diff
@@ -0,0 +1,54 @@
+{% set name = "bm3d" %}
+{% set version = "3.0.9" %}
+
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/bm3d-{{ version }}.tar.gz
+  sha256: ca6e416f9ec491ca3c408ad796efd3115479177998d50b2c1a2f83fda75648ef
+
+build:
+  number: 0
+  noarch: python
```

Comment:
For starters, this can't be a `noarch: python` package if you're building and including C extensions for Python.

Reply:
Thanks; removed it.